modules/@apostrophecms/asset/index.js

```diff
@@ -48,26 +56,17 @@ module.exports = {
     self.extraBundles = fillExtraBundles(verifiedBundles);
     self.webpackExtensions = extensions;
     self.verifiedBundles = verifiedBundles;
+    self.buildWatcherDisable = !!process.env.APOS_ASSETWATCH_DISABLE || (self.options.watch === false);
```

Comment:
Check `APOS_ASSET_WATCH` for the string `0` instead. I try to avoid options phrased in the negative.